pkg/testutil/e2eutil/sysprocattr.go

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) The Thanos Authors.
 // Licensed under the Apache License 2.0.
 
+//go:build !linux
```

Comment:
This seems unrelated to the current changes.

Reply:
yeah that happens when I ran the command `make assets` it called `make format` and this one is doing this.
Maybe it's a missed when moving to golang v1.17?